.env.example

```diff
@@ -1,23 +1,23 @@
 APP_NAME=Laravel
 APP_ENV=local
-APP_KEY=
+APP_KEY=base64:hda6MjikxoMqfPAK4uSHvo0z6QIhfpxKShfytQe/3bo=
 APP_DEBUG=true
 APP_URL=http://localhost:8000
 
 LOG_CHANNEL=stack
 
 DB_CONNECTION=mysql
-DB_HOST=127.0.0.1
+DB_HOST=database
```

Comment:
Shouldn't the host be the name or IP address of a host computer?

Reply:
I think you can define those environment variables to have a list of acceptable hosts

Reply:
The docker compose file defines the service as database. so it recognises the service by name instead of ipaddress since it is dynamic.

Reply:
Oh, I see now.